app/models/rating.rb

```diff
@@ -3,6 +3,7 @@ class Rating < ActiveRecord::Base
   belongs_to :user
 
   scope :for_event, -> (event) { joins(:proposal).where("proposals.event_id = ?", event.id) }
+  scope :not_withdrawn, -> { joins(:proposal).where("proposals.state != ?", WITHDRAWN) }
```

Comment:
You've got the right idea, but I don't think this will quite work. When load the teammate page, I get an error:
  uninitialized constant Rating::WITHDRAWN